src/test/scss/tools/mixins/type.spec.scss

```
@@ -0,0 +1,39 @@
+@use 'true' as *;
+@use 'src/scss/tools/mixins/type' as systemUnderTest;
+
+@include describe('rem [mixin]') {
+    @include it('outputs the expected CSS by rounding `$sizeValue`') {
+        // arrange
+        $property: padding;
+        $sizeValue: 10.5;
+
+        // act & assert
+        @include assert {
+            @include output {
+                @include systemUnderTest.rem($property, $sizeValue);
+            }
+
+            @include expect {
+                padding: 11px;
+                padding: 0.65625rem
+            }
+        }
+    }
+
+    @include it('outputs the expected CSS by rounding the default `$sizeValue` when no `$sizeValue` argument is provided') {
```

This one isn't rounding, right?
```suggestion
    @include it('outputs the expected CSS using the default `$sizeValue` when no `$sizeValue` argument is provided') {
```